File: sandbox.client/front/ts/model/ClientPhone.ts
import {PhoneType} from "./PhoneType";

export class ClientPhone {
  public client:number;
  public type:PhoneType;
  public number:string;
  public name:string;

  constructor(){
    this.type=PhoneType.MOBILE;

  }
  public assign(o:any):ClientPhone{
    this.type=o.type;
    this.number=o.number;
    this.name=o.name;
    return this;
  }
}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/model/ClientPhone.java
package kz.greetgo.sandbox.controller.model.model;



public class ClientPhone {
    public Integer client;
    public PhoneType type;
    public String number;

    @Override
    public String toString() {

        return "ClientPhone{" +
          "client=" + client +
          ", type=" + type +
          ", number='" + number + '\'' +
          '}';
    }
}
